src/core/CoolBoard.cpp

```diff
@@ -658,7 +658,7 @@ void CoolBoard::updateFirmware(String firmwareVersion, String firmwareUrl,
     delete this->coolWifi;
     INFO_LOG("Starting firmware update...");
     t_httpUpdate_return ret =
-        ESPhttpUpdate.update(firmwareUrl, "", firmwareUrlFingerprint, true);
+        ESPhttpUpdate.update(firmwareUrl, "", firmwareUrlFingerprint);
```

Comment:
What does it changes ?

Reply:
true is a boolean to indicate if need to reboot or not after the ota update, is true for default